lib/display.py

```diff
@@ -146,6 +171,7 @@ def main():
         data_constructor.build_data()
         data = load_data()
     data["want_to_play"] = True
+    prev_start = data.get("start", None)
     data["start"] = time.time()
```

Comment:
this line now concerns me - because really this update should be happening in lib/update when we're going through all of the actions for the time interval since last play
It makes me think that there should be a better method for getting the time interval instead of generating the delta both in display and in update